Description: Use build flags from environment (dpkg-buildflags).
 Necessary for hardening flags.
 .
 *_BUILD flags don't contain the flags from the environment, but only local
 additions.
Index: lynx-cur-2.8.8dev.12/src/chrtrans/makefile.in
===================================================================
--- lynx-cur-2.8.8dev.12.orig/src/chrtrans/makefile.in	2012-03-19 16:50:15.072775970 +0100
+++ lynx-cur-2.8.8dev.12/src/chrtrans/makefile.in	2012-03-19 16:51:46.744779460 +0100
@@ -123,7 +123,7 @@
 C_SRC		= $(OBJS:$o=.c)
 
 $(MAKEUCTB) : $(OBJS)
-	$(BUILD_CC) $(CC_OPTS) $(BUILD_LDFLAGS) -o $@ $(OBJS) $(INTLLIB) $(BUILD_LIBS)
+	$(BUILD_CC) $(CC_OPTS) $(LDFLAGS) $(BUILD_LDFLAGS) -o $@ $(OBJS) $(INTLLIB) $(BUILD_LIBS)
 
 makeuctb$o : $(srcdir)/UCkd.h $(srcdir)/makeuctb.c
 
